samples/language/Comments.qs

```diff
@@ -0,0 +1,21 @@
+/// # Sample
+/// Comments
+///
+/// # Description
+/// Comments begin with two forward slashes (//) and continue until the
+/// end of line. Such end-of-line comments may appear anywhere in the
+/// source code. Q# does not currently support block comments.
+/// Documentation comments, or doc comments, are denoted with three
+/// forward slashes instead of two.
+
```

Comment:
nit: I think the blank line between doc comments and the namespace declaration is not needed.

Reply:
The same applies to the other files in this PR.